src/main/java/somethingrandom/interfaceadapters/additem/AddItemView.java

```diff
@@ -0,0 +1,143 @@
+    private static JFrame frame = new JFrame();
+    private static JLabel itemTitle = new JLabel();
+    private static JTextField itemtitleInput = new JTextField(100);
+    private static JLabel itemDisc = new JLabel();
+    private static JTextField itemDiscInput = new JTextField(10);
+    private static JButton saveItem;
+    private final AddItemController itemController;
+    private AddItemViewModel addItemViewModel;
+
+
+    public AddItemView(AddItemController itemController, AddItemViewModel addItemViewModel) {
+        this.itemController = itemController;
+        this.addItemViewModel = addItemViewModel;
+
+        JLabel title = new JLabel("ADD ITEM");
+        title.setAlignmentX(Component.CENTER_ALIGNMENT);
+
+        JPanel buttons = new JPanel();
+        saveItem = new JButton();
+        buttons.add(saveItem);
+
+        saveItem.addActionListener(
+            new ActionListener() {
+                @Override
+                public void actionPerformed(ActionEvent e) {
+                    if (e.getSource().equals(saveItem)){
+                        AddItemState currentState = addItemViewModel.getState();
+
+                        itemController.execute(
+                            currentState.getItemName(),
+                            currentState.getDescription()
+                        );
+                    }
+                }
+            }
+        );
+        itemtitleInput.addKeyListener(new KeyListener() {
+            @Override
+            public void keyTyped(KeyEvent e) {
+                AddItemState currentState = addItemViewModel.getState();
+                currentState.setItemName(itemtitleInput.getText() + e.getKeyChar());
+                addItemViewModel.setState(currentState);
+
+            }
+
+            @Override
+            public void keyPressed(KeyEvent e) {
+
+            }
+
+            @Override
+            public void keyReleased(KeyEvent e) {
+
+            }
+        });
+        itemDiscInput.addKeyListener(
+            new KeyListener() {
+                @Override
+                public void keyTyped(KeyEvent e) {
+                    AddItemState currentState = addItemViewModel.getState();
+                    currentState.setItemName(itemDiscInput.getText() + e.getKeyChar());
+                    addItemViewModel.setState(currentState);
+
+                }
+
+                @Override
+                public void keyPressed(KeyEvent e) {
+
+                }
+
+                @Override
+                public void keyReleased(KeyEvent e) {
+
+                }
+            }
+        );
+
+    }
+
+
+
+    public static void main(String[] args) {
+        frame.setTitle("Add Item");
+        frame.setSize(600,400);
+        frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+        frame.add(panel);
+
+        panel.setLayout(null);
+        itemTitle= new JLabel("Item:");
+        itemTitle.setBounds(50, 40, 80, 25);
```

Comment:
`setBounds` is probably going to make the interface really fragile, since resizing requires changing all the coördinates. It might be OK for now, but it might be worth investigating using a [`GridLayout` for the panel](https://docs.oracle.com/javase/tutorial/uiswing/layout/grid.html), or something like that.

Right now, resizing looks like this: 
![image](https://github.com/nnzahin/Brainsweep/assets/30192076/405adb29-7188-44bd-9cb7-41223d6c0b92)